Behavior/Resources/Localizable.xcstrings

```diff
@@ -524,13 +528,13 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "This application demonstrates several Spezi features & modules."
+            "value" : "This application collects data about your step count and active energy burned."
           }
         }
       }
     },
     "WELCOME_TITLE" : {
-      "comment" : "MARK: Welcome",
+      "comment" : "MARK: Welcome to the Behavior Application!",
```

Comment:
Was it your intention to edit the comment in Xcode? I think you might wanted to edit the value of the localization string?

If you want to learn more about localization, please check out this video for more details: https://developer.apple.com/wwdc23/10155.